Tasks/InvokeRestApi/task.json

```diff
@@ -58,12 +58,12 @@
       "name":"headers",
       "type":"multiLine",
       "label":"Headers",
-      "required":"false",
-      "defaultValue":"{\n\"Content-Type\":\"application/json\"\n}",
+      "required":"true",
```

Comment:
why it is mandatory?

Reply:
We're filling task* related data in headers, can be used to update task timeline record from customer's external service.